hack/update-ends-newline.sh

```diff
@@ -20,14 +20,7 @@ set -o pipefail
 ROOT_DIR=$(dirname "${BASH_SOURCE[0]}")/..
 
 function update_ends_newline() {
-  find . \
-    -iname "*.md" \
-    -o -iname "*.sh" \
-    -o -iname "*.go" \
-    -o -iname "*.tpl" \
-    -o -iname "*.yaml" \
-    -o -iname "*.yml" |
-    xargs -I {} bash -c "[ -n \"\$(tail -c 1 {})\" ] && echo >> {}" || :
+  find . \( -iname "*.md" -o -iname "*.sh" -o -iname "*.go" -o -iname "*.tpl" -o -iname "*.yaml" -o -iname "*.yml" \) -exec sh -c '[ -n "$(tail -c 1 "$1")" ] && echo "$1"' sh {} \;
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2038

Should not be combined in one line